player_darwin.c

```diff
@@ -0,0 +1,143 @@
+//
+// Created by Christopher Cooper on 7/12/17.
+//
```

Comment:
Sure thing - CLion does that when you create files, I didn't even realize it was there